### DISCUSSION TITLE
When will mixer_tts pretrained model added?

### DISCUSSION THREAD

**Discussion by yuananf** [NVIDIA-NeMo/NeMo#3320] [Q&A] [answered]
I'd like to test it when it's possible. As said in the paper, it has a small number of parameters and enable much faster speech synthesis compared to the models with similar quality.

**Comment by XuesongYang** [comment, marked as answer]
mixer_tts pretrained model checkpoint has already been released. Please try,
```
from nemo.collections.tts.models import MixerTTSModel

print(MixerTTSModel.list_available_models())
```

you will find `"tts_en_lj_mixertts"` which is the one you are interested in. We have a tutorial on how to train mixer_tts as well: https://github.com/NVIDIA/NeMo/blob/main/tutorials/tts/FastPitch_MixerTTS_Training.ipynb
